### PR TITLE
PCLOUDS-3136 correct linking of logs

### PR DESCRIPTION
**Pull request by PatrykOzogDynatrace**

### CODE REVIEW

File: logs_ingest/mapping.py [hunk 1]
```diff
@@ -49,7 +49,12 @@
             resource_type = resource_type_to_me_type["resourceType"].lower()
             category = resource_type_to_me_type.get("category", "").lower()
             key = ",".join(filter(None, [resource_type, category]))
-            dt_me_type_mapper.update({key: resource_type_to_me_type["meType"]})
+            me_type = resource_type_to_me_type["meType"]
```
Comment (mateuszadamczakDynatrace) [inline logs_ingest/mapping.py:52]: [optional] I think metric_type would be more readable than me_type

File: tests/unit/test_dynatrace_client.py [hunk 2]
```diff
@@ -74,6 +74,7 @@ def test_prepare_serialized_batches_split_by_events_limit(monkeypatch: MonkeyPat
     logs = [create_log_entry_with_random_len_msg() for x in range(how_many_logs)]
 
     batches = dynatrace_client.prepare_serialized_batches(logs)
+    print(batches)
```
Comment (mateuszadamczakDynatrace) [inline tests/unit/test_dynatrace_client.py:77]: please remove this print

File: logs_ingest/dynatrace_client.py [hunk 3]
```diff
@@ -19,6 +19,7 @@
 import time
 import aiohttp
 import asyncio
+import ast
```
Comment (mateuszadamczakDynatrace) [inline logs_ingest/dynatrace_client.py:22]: so how does this work? Is there a way to avoid using ast library?

Comment (PatrykOzogDynatrace) [reply logs_ingest/dynatrace_client.py:22]: Well the ast is used to convert a string looking as list to list. The ast.literal_eval is created specifically for that kind of operations. Of course, I can walk-around this library but other solutions might break is some corner cases.

Comment (PatrykOzogDynatrace) [reply logs_ingest/dynatrace_client.py:22]: Well I use ast.literal_eval to convert a string looking as list to a list. Of course, I can walk-around this library but other solutions might break is some corner cases.

File: logs_ingest/mapping.py (hunk 1 above)
Comment (NematulloKozimov) [reply logs_ingest/mapping.py:52]: me = monitored entity

File: logs_ingest/dynatrace_client.py (hunk 3 above)
Comment (mateuszadamczakDynatrace) [reply logs_ingest/dynatrace_client.py:22]: Leave it like that, since it built into python it's fine for me

File: logs_ingest/mapping.py (hunk 1 above)
Comment (mateuszadamczakDynatrace) [reply logs_ingest/mapping.py:52]: As you can see there's a problem with naming here

**Review by mateuszadamczakDynatrace — APPROVED** [review]

**Pull request closed without merging**